requirements.txt

```diff
@@ -1,4 +1,5 @@
 requests
 prettytable
 python-dateutil
-tox
\ No newline at end of file
+.
```

Comment:
why `.`?

Reply:
It's same as `pip install git-stalk`.

This way we install git-stalk project in travis cli, so we can use terminal commands in test_user_existence.py.